src/hardware/portal/index.ts

```diff
@@ -194,7 +195,11 @@ export const signPSBT = (psbt: string) => {
 };
 
 export const getXpub = ({ isMultisig = true }) => {
-  const derivationPath = isMultisig ? 'm/48h/1h/0h/2h' : 'm/84h/1h/0h';
+  const derivationPath = isMultisig
+    ? isTestnet()
+      ? 'm/48h/1h/0h/2h'
+      : 'm/48h/0h/0h/2h'
+    : 'm/84h/1h/0h';
```

Comment:
This is still using testnet always for the single sig. It should also use the 0 instead of 1 for mainnet

Reply:
updated